doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -66,6 +66,17 @@ Build and configuration system
      This feature has never been functional.
      To configure the signing key, use any available Kconfig method.
 
+* Deprecated the following devicetree properties:
+
+  * ``owner-id``
+  * ``perm-read``
+  * ``perm-write``
+  * ``perm-execute``
+  * ``perm-secure``
+  * ``non-secure-callable``
+
+  It is recommended to replace them with a new devicetree property: ``nordic,access``.
+  See the :ref:`migration guide <migration_2.8_recommended>` for more information.
```

Comment:
```suggestion
  See the :ref:`migration guide <migration_2.8_devicetree>` for more information.
```

Reply:
That seemed inconsistent to me. Lately, links like this have not been that granular:
https://github.com/nrfconnect/sdk-nrf/blob/3f0f5e710a6b904924af3f21a58ca2b7dc4f76ad/doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst?plain=1#L809
https://github.com/nrfconnect/sdk-nrf/blob/3f0f5e710a6b904924af3f21a58ca2b7dc4f76ad/doc/nrf/releases_and_maturity/releases/release-notes-2.7.0.rst?plain=1#L167

Reply:
Hmm. Not how I'd do it, but you are right, it is how we've done it in the past. Alright, that part is ok.